ui-spacetimechart/src/types.ts

```diff
@@ -0,0 +1,15 @@
+export type SpaceTimeRegion = {
+  type?: string;
+  timeStart: Date; // timestamp
+  timeEnd: Date; // timestamp
+  spaceStart: number; // mm
+  spaceEnd: number; // mm
+  image?: string;
+};
+
+export type WorkSchedule = {
+  type: 'TRACK' | 'CATENARY';
+  timeStart: Date;
+  timeEnd: Date;
+  spaceRanges: [number, number][];
```

Comment:
Why spaceRange is `[number, number][]` and not `[number, number]` ?

Reply:
you have several portions blocked for the same period of time. if the track goes from 0 to 100 for example, you can have [[10,20], [50, 80]]

Reply:
Ok clear, thanks :pray: